Marcio/registrar content strategy update (#2363)
cloudflare/cloudflare-docs: products/registrar/src/content/faq.md

pcx-content-type: faq
---

# FAQ

Below you will find answers to our most commonly asked questions on Cloudflare Registrar, namely about billing, domain transfers, and how to troubleshoot common issues you may come across. If you cannot find the answer you are looking for, heading over to our [community page](https://community.cloudflare.com/) is always a good idea.

* [Domain transfers](#domain-transfers)
* [Domain registrations](#domain-registrations)
* [Billing](#billing)

## Domain transfers

### Why did my transfer fail?

Domain transfers sometimes fail. Your account team can help you address what failed so that you can complete the transfer to Cloudflare.

1. Make sure that you entered the auth code for that domain. If you need to reenter it, you can do so from your **Account Home** > **Registrar** > **Transfer** page.
1. You can retry the transfer from the transfer launch page.

If you cannot solve the issue, open a support ticket or contact your account team.

> ## Why did my domain's expiration date changed after transferring it to Cloudflare?

Reviewer: Should this be a level-3 heading also, to be included in the "Domain transfers" section?

> # Frequently Asked Questions

Reviewer: ```suggestion
# FAQ
``` 
According to our content strategy, the title should be "FAQ". The same for the sidebar title.

> Below you will find answers to our most commonly asked questions on Cloudflare Registrar, how billing works, transfers and how to troubleshoot common issues you may come across. If you cannot find the answer you are looking for, heading over to our [community page](https://community.cloudflare.com/) is always a good idea.

Reviewer: ```suggestion
Below you will find answers to our most commonly asked questions on Cloudflare Registrar, namely about billing, domain transfers, and how to troubleshoot common issues you may come across. If you cannot find the answer you are looking for, heading over to our [community page](https://community.cloudflare.com/) is always a good idea.
```

> * [Domain transfers](#why-did-my-transfer-fail)

Reviewer: Any reason for highlighting these three specific entries? They're not categories.

Author: Just because in our content strategy it states that for medium FAQs (more than 10) we should structure it into sections to facilitate readability and discoverability of content.

Reviewer: When using sections, I think it should link to a section and not to a specific question. In this case, section names would be level-2 headings, and questions would become level-3 headings.

Author: Should be addressed

* [Domain registrations](#can-i-register-a-premium-domain)
* [Billing](#when-will-i-be-billed)

## Why did my transfer fail?

Domain transfers sometimes fail. Your account team can help you address what failed so that you can complete the transfer to Cloudflare.

1. Make sure that you entered the auth code for that domain. If you need to reenter it, you can do so from your **Account Home** > **Registrar** > **Transfer** page.

> 1. You can reattempt the transfer from the transfer launch page.

Reviewer: Maybe "retry" instead of "reattempt" (unless it's what appears in the UI)?

Author: done

If you cannot solve the issue, open a support ticket or contact your account team.

## Why did my domain's expiration date changed after transferring it to Cloudflare?

> ICANN requires that any transfer also extends the expiration date of your domain by at least one year — that is one year from your current expiration date, not one year from the date of transfer. For example, if you transfer a domain on October 10, 2020, but it expires on March 10, 2021, your new expiration date will be March 10, 2022.

Reviewer: ```suggestion
ICANN requires that any transfer also extends the expiration date of your domain by at least one year — that is one year from your current expiration date, not one year from the date of transfer. For example, if you transfer a domain on October 10, 2021, but it expires on March 10, 2022, your new expiration date will be March 10, 2023.
```
Expiration should be in the future, so I suggest that we update the dates to 2021-2023.

Once successful, you will receive an email from Cloudflare and be able to manage the domain in the dashboard under **Overview** of that site.

## Why am I not allowed to transfer my domain?

ICANN prohibits domain transfers within 60 days of a change to the WHOIS data or registrar of a domain. If you modified your contact information, transferred registrars, or registered your domain in the last 60 days, Cloudflare will be unable to process your transfer immediately.

You can leave the domain **In Progress** and Cloudflare will attempt to process the transfer until the 60-day window passes.

> 

Reviewer: We don't need this empty line.

Author: done

## Why am I not able to start a transfer?

If you have an <a href='https://support.cloudflare.com/hc/articles/203471284#h_1l0KGygoBX9QYjNrhAcHjg'>unverified email address</a>, you might experience issues when initiating a domain transfer.

> ## What happens if I entered the wrong auth code?

Reviewer: ```suggestion
## What happens if I enter the wrong auth code?
```
Alternatively, "What should I do if I entered an incorrect auth code?"

> If you input the incorrect auth code (also referred to as authentication code and authorization code), return to the **Domain Registration** page or the **Overview** for your site. You can use the available input field to reenter your authentication code.

Reviewer: ```suggestion
If you enter an incorrect auth code (also referred to as authentication code or authorization code), return to the **Domain Registration** page or the **Overview** for your site. You can use the available input field to reenter your authentication code.
```

---

## Can I register a premium domain?

Cloudflare does not currently support premium domains. Some registries designate a domain name as “premium” and charge higher wholesale rates for these domains.

## My domain’s registration was not extended by one year after transferring to Cloudflare

When you transfer your domain to Cloudflare, the registry will extend your registration by one year. However, in one specific circumstance your transfer could result in you keeping your original expiration date.

When a domain expires, the registration enters the auto-renew grace period. During that time, you can renew the domain at your registrar to avoid losing it. If your domain expires at your current registrar, you renew it and then transfer to Cloudflare within 45 days, the registry can restrict the addition of an extra year.

> Say you have `example.com` registered and it expires on December 10, 2020. You decide to renew it during the auto-renew grace period on December 20, 2020. That renewal extends the registration to December 20, 2021. You then transfer to Cloudflare on December 30, 2020. Since that transfer is within 45 days of the expiration, the registry may not add the year to your registration. When you transfer to Cloudflare or any registrar in this circumstance, your expiration can still remain December 20th, 2021.

Reviewer: ```suggestion
Say you have `example.com` registered and it expires on December 10, 2021. You decide to renew it during the auto-renew grace period on December 20, 2021. That renewal extends the registration to December 20, 2022. You then transfer to Cloudflare on December 30, 2021. Since that transfer is within 45 days of the expiration, the registry may not add the year to your registration. When you transfer to Cloudflare or any registrar in this circumstance, your expiration can still remain December 20th, 2022.
```

If a year is not added to your registration, you have effectively paid twice for the same added year. Per ICANN rules, you are entitled to request a refund at your previous registrar.

## What Happens When a Domain Expires?

> Cloudflare currently offers a 40 day grace period for most top-level domains (TLDs). During this period you may renew/extend the domain at any time from within the dashboard but no further auto-renew attempts will be made. For the first 30 days of the grace period, the domain will continue to resolve as normal. On the 30th day after the expiration date, the domain will be suspended and a parked suspension page will be displayed. You may still renew the domain at any time during this suspension period. On the 40th day, the domain will enter the Redemption Period and will no longer resolve to any web page. The redemption period lasts for 30 days.  During this time, it may be possible to restore and renew the domain. A restore fee may apply in addition to the renewal fee. Contact your account team for assistance. At the end of the 30 day redemption period, the domain will be placed in pending delete status for a period of five days, after which it will be released and made available for re-registration. The domain cannot be restored or renewed during this period.

Reviewer: ```suggestion
Cloudflare currently offers a 40-day grace period for most top-level domains (TLDs). During this period you may renew/extend the domain at any time from within the dashboard but no further auto-renew attempts will be made. For the first 30 days of the grace period, the domain will continue to resolve as normal. On the 30th day after the expiration date, the domain will be suspended and a parked suspension page will be displayed. You may still renew the domain at any time during this suspension period. On the 40th day, the domain will enter the Redemption Period and will no longer resolve to any web page. The redemption period lasts for 30 days.  During this time, it may be possible to restore and renew the domain. A restore fee may apply in addition to the renewal fee. Contact your account team for assistance. At the end of the 30 day redemption period, the domain will be placed in pending delete status for a period of five days, after which it will be released and made available for re-registration. The domain cannot be restored or renewed during this period.
```
(Just added a hyphen in `40-day`.)

* **Day 1 - 30:** Grace Period (domain resolves normally).
* **Day 31 - 40:** Suspension Period (domains resolves to suspension page).
* **Day 41 - 70:** Redemption Period.
* **Day 71 - 75:** Pending Delete Period.

---

## When will I be billed?

> You will be billed when you input your authorization code and initiate the transfer of your domain to Cloudflare. Make sure to copy / paste the code to avoid mistakes. The transfer will not initiate if the code is incorrect.

Reviewer: ```suggestion
You will be billed when you input your authorization code and initiate the transfer of your domain to Cloudflare. Make sure to copy and paste the code to avoid mistakes. The transfer will not initiate if the code is incorrect.
```
